src/course-outline/utils.jsx

```diff
@@ -109,8 +109,16 @@ const getHighlightsFormValues = (currentHighlights) => {
   return formValues;
 };
 
+/* eslint no-param-reassign: "error" */
+const scrollToBottom = (ref) => {
```

Comment:
Why do we need this eslint comment?

Also renaming this util to something like `scrollToElement` would be better no?

Reply:
We don't need the eslint comment here, I'm removing it.

And yes, I agree is best to rename the util to `scrollToElement`.